kratos/utilities/variable_utils.h

```diff
@@ -1728,7 +1728,7 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         NodeType& rNode,
         const int Step = 0)
     {
-        if constexpr(std::is_integral_v<TDataType> || std::is_floating_point_v<TDataType>) {
+        if constexpr(std::is_integral_v<TDataType> || std::is_floating_point_v<TDataType> || std::is_same_v<TDataType, Matrix> || std::is_same_v<TDataType, Vector>) {
```

Comment:
but `Matrix` and `Vector` should use `noalias` no? 🤔

Reply:
Thats what i thought too, but they fail if they are not properly initialized in the first place with the error bad index

Reply:
`noalias` would be useful if we could guarantee that the object on the left doesn't appear in the expression on the right. However, both are inputs in this case so we can't guarantee anything => the branch with `noalias`  should be deleted and we should always assign without `noalias`.

Btw this is a simple assignment; there's no point in using `noalias` because there's no intermediate step here, no potential temporary objects.